roxie/ccd/ccdprotocol.cpp

```diff
@@ -242,9 +243,16 @@ class ProtocolSocketListener : public ProtocolListener
         keyFile.set(_keyFile);
         passPhrase.set(_passPhrase);
         isSSL = streq(protocol.str(), "ssl");
+        DBGLOG("ProtocolSocketListener port=%d, certFile=%s, keyFile=%s", port, _certFile, _keyFile);
```

Comment:
Double checking that this is only logged once for the process.

Reply:
Yes, these should be instantiated (and log) at roxie startup time.

Reply:
That being said, I don't think I meant to add this line, it doesn't even cover my change, I'll remove it.

Reply:
Removed.